sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -207,6 +207,10 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
 
     case ShowCurrentNamespaceStatement() =>
       ShowCurrentNamespace(catalogManager)
+
+    case ShowTablePropertiesStatement(NonSessionCatalog(catalog, tableName), propertyKey) =>
+      ShowTblproperties(catalog.asTableCatalog, tableName.asIdentifier, propertyKey)
```

Comment:
The logical plan should take a `NamedRelation`, and here we just pass in a `UnresolvedV2Relation`.

You can take a look at how `DescribeTableStatement` is handled in this file.

Reply:
Ok, I see, the Analizer resolve tables in V2 catalog, thanks!!